### PR TITLE
Feat huevos con salchicha

### PR DESCRIPTION
**Pull request by abrilmunozc**

### CODE REVIEW

File: 04/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        2 Cucharadas de Jugo MAGGI®
+        6 Huevos ligeramente batidos
+      </ul>
+    </section>
+  <aside>Pasos para preparar el platillo</aside>
+    <article>
+      <ol>
+        Calienta el aceite en una sartén, agrega las salchichas y fríelas un poco, sazónalas con el Jugo MAGGI®.
+        Vierte los huevos, mezcla y cocina hasta que cuajen. Ofrece
+      </ol>
```
Comment (eamzea) [inline 04/index.html:29]: Where's the list?

File: 04/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta description="Receta para preparar huevos con salchicha">
+  <meta keywords="receta, huevos con salchicha, huevos, salchicha">
+  <title>Receta para preparar huevos con salchicha</title>
+</head>
+<body>
+  <main>
+    <p>Huevos con salchicha</p>
+  </main>
+  <header>Receta para preparar huevos con salchicha</header>
```
Comment (eamzea) [inline 04/index.html:14]: Header is a text tag?

File: 04/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <article>
+      <ol>
+        Calienta el aceite en una sartén, agrega las salchichas y fríelas un poco, sazónalas con el Jugo MAGGI®.
+        Vierte los huevos, mezcla y cocina hasta que cuajen. Ofrece
+      </ol>
+    </article>
+  <footer>
+    Consejo nutritional
+    Utiliza salchichas con bajo contenido de sodio.
+  </footer>
```
Comment (eamzea) [inline 04/index.html:34]: Footer is a text tag?

File: 04/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  <nav>Ingredientes:</nav>
+    <section>
+      <ul>
+        2 Cucharadas de Aceite de maíz
+        3 Salchichas cortadas en rodajas
+        2 Cucharadas de Jugo MAGGI®
+        6 Huevos ligeramente batidos
+      </ul>
+    </section>
+  <aside>Pasos para preparar el platillo</aside>
```
Comment (eamzea) [inline 04/index.html:24]: Aside is a text tag?

File: 04/index.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  <meta description="Receta para preparar huevos con salchicha">
+  <meta keywords="receta, huevos con salchicha, huevos, salchicha">
+  <title>Receta para preparar huevos con salchicha</title>
+</head>
+<body>
+  <main>
+    <p>Huevos con salchicha</p>
+  </main>
+  <header>Receta para preparar huevos con salchicha</header>
+  <nav>Ingredientes:</nav>
```
Comment (eamzea) [inline 04/index.html:15]: Nav is a text tag?

File: 04/index.html
Comment (eamzea) [inline 04/index.html:1]: Indentation

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 04/index.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  <meta description="Receta para preparar huevos con salchicha">
+  <meta keywords="receta, huevos con salchicha, huevos, salchicha">
+  <title>Receta para preparar huevos con salchicha</title>
+</head>
+<body>
+  <main>
+    <p>Huevos con salchicha</p>
+    <header>
+    <p>Receta para preparar huevos con salchicha</p>
+    </header>
```
Comment (eamzea) [inline 04/index.html:15]: Indentation

**Review by soyedgarzea — CHANGES_REQUESTED** [review]
Branch is outdated

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**